### PR TITLE
[PLT-7781] Set up argo-workflows

### PR DESCRIPTION
**Pull request by shlevy**
## Summary by CodeRabbit

- **New Features**
	- Added `kustomize` package to the cloud development environment.
	- Introduced a new Terraform resource "kubectl_manifest" for managing Kubernetes objects.
- **Refactor**
	- Changed the Terraform source in the production environment from a GitHub repository to a local path.

### CODE REVIEW

File: infra/us-east-1/prod/eks/blue/eks-addons/terragrunt.hcl [hunk 1]
```diff
@@ -21,7 +21,7 @@ include {
 }
 
 terraform {
-  source = "github.com/input-output-hk/sc-dev-platform.git//infra/modules/eks/addons"
+  source = "../../../../../modules/eks/addons"
```
Comment (shlevy) [inline infra/us-east-1/prod/eks/blue/eks-addons/terragrunt.hcl:24]: Why were we re-fetching from GH?

**Comment by shlevy** [comment]
Note: This is applied with local diff for testing:

```diff
diff --git a/infra/modules/eks/addons/manifests/kube-objects.yaml b/infra/modules/eks/addons/manifests/kube-objects.yaml
index a2e8ded..506876f 100644
--- a/infra/modules/eks/addons/manifests/kube-objects.yaml
+++ b/infra/modules/eks/addons/manifests/kube-objects.yaml
@@ -10,7 +10,7 @@ spec:
       type: kustomize
       properties:
         git:
-          branch: master
+          branch: argo-setup
           provider: GitHub
         path: infra/kube-objects
         prune: true

```

File: infra/kube-objects/kustomization.yaml
Comment (shlevy) [inline infra/kube-objects/kustomization.yaml:1]: Note that we can use this Kustomize directory for declarative k8s config without terraform (using kubevela + fluxcd)

File: infra/us-east-1/prod/eks/blue/eks-addons/terragrunt.hcl (hunk 1 above)
Comment (tshaynik) [reply infra/us-east-1/prod/eks/blue/eks-addons/terragrunt.hcl:24]: This was something we had considered but that didn't work well, and were moving away from. We had already reverted it in most places, so it's good that it's reverted here as well.

**Review by tshaynik — APPROVED** [review]
This is cool! It's great that manifests added to `infra/kube-obects` will be automatically deployed!

**Pull request merged**